Source/WebCore/css/CSSProperties.json

```diff
@@ -6012,14 +6012,31 @@
                     "transition-property",
                     "transition-duration",
                     "transition-timing-function",
-                    "transition-delay"
+                    "transition-delay",
+                    "transition-behavior"
                 ]
             },
             "specification": {
                 "category": "css-transitions",
                 "url": "https://www.w3.org/TR/css3-transitions/#transition-shorthand-property"
             }
         },
+        "transition-behavior": {
+            "values": [
+                "normal",
+                "allow-discrete"
+            ],
+            "animatable": true,
+            "codegen-properties": {
```

Comment:
Might be worth adding a setting unless you're 100% sure you can get this done soon :)

Reply:
I have the followup patch that completes the feature ready to go, so I don't think it will be warranted.